src/hotspot/share/oops/objArrayKlass.hpp

```diff
@@ -78,7 +82,6 @@ class ObjArrayKlass : public ArrayKlass {
   static ObjArrayKlass* allocate_objArray_klass(ClassLoaderData* loader_data,
                                                 int n, Klass* element_klass, TRAPS);
 
-  objArrayOop allocate(int length, TRAPS);
```

Comment:
Do you think `multi_allocate` will need a better name in the future?

Reply:
I thought if changing multi_allocate_instance so that it's clear that it's an instance, but decided to limit this.  Maybe this would be helpful but the allocate() function was the most confusing to me, that's why I picked that one.